yi-core/yi-core.cabal

```diff
@@ -135,3 +136,54 @@ library
       System.CanonicalizePath
       Yi.Buffer.Implementation
       Paths_yi_core
+
+test-suite tasty
```

Comment:
I just ran stack build, stack test, stack exec hpack and hpack without stack (I removed the .cabal file inbetween) and they all give me the same result. My hpack version is `0.14.1`.